parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -787,6 +774,15 @@ loop = do
                 (foldMap SR'.labeledDependencies $ failed <> failedDependents)
             respond $ CantDelete input ppe failed failedDependents
 
+      DisplayI outputLoc s@(HQ.unsafeFromString -> hq) -> do 
+        parseNames <- (`Names3.Names` mempty) <$> basicPrettyPrintNames0
+        let results = Names3.lookupHQTerm hq parseNames 
+        if Set.null results then 
+          respond $ SearchTermsNotFound $ [hq]
+        else if Set.size results > 1 then
+          respond $ TermAmbiguous input (Right hq) results
+        else doDisplay outputLoc parseNames (Set.findMin results)
```

Comment:
Note - moved most of `display` logic out to `doDisplay` function, which is used by both `display` and `docs` commands.